src/pages/docs/runbooks/config-as-code-runbooks.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2024-11-24
+modDate: 2024-11-24
+title: Config as Code runbooks
+description: Details about using configuration as code with runbooks.
+navOrder: 20
+icon: fa-brands fa-git-alt
+---
+
+:::div{.success}
+Support for CaC Runbooks is rolling out to Octopus Cloud now and will be available in Octopus Server in a future release
+:::
+
+Config as Code (or CaC) Runbooks stores your runbook process as code in your project repository. This means that you can now use version control to track changes to your runbook processes alongside changes to your project code.
+
+You may already be using CaC to store your Octopus deployment process, deployment settings and non-sensitive variables. Adding CaC Runbooks completes that picture. 
+
+The config for your runbooks, as well as your deployments, settings and variables, is stored as OCL files which you can edit in Octopus or directly in your IDE. 
+
+Using CaC Runbooks is optional. You can also choose to keep using Runbooks as you always have on your existing version controlled projects and on projects without version control.
```

Comment:
It's optional for now only because we don't want to forcible migrate. It might be clearer to talk about how the _migration_ is currently optional.